.github/actions/browserslist-update/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Update Browserslist database
+
+on:
+    schedule:
+        - cron: '0 0 5,15,25 * *'
+    workflow_dispatch:
+
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    update-browserslist-database:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+            - name: Configure git
+              run: |
+                  # Setup for committing using built-in token. See https://github.com/actions/checkout#push-a-commit-using-the-built-in-token
+                  git config user.name "github-actions[bot]"
+                  git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+            - name: Update Browserslist database and create PR if applies
+              uses: c2corg/browserslist-update-action@v2
+              with:
+                  base_branch: main
+                  commit_message: 'build: update Browserslist db'
+                  title: 'build: update Browserslist db'
+                  body: Auto-generated by [browserslist-update-action](https://github.com/c2corg/browserslist-update-action/)
```

Comment:
Should we always assign someone to that PR to guarantee it gets reviewed eventually? What's the best team to assign this to?

Reply:
there's an automerge label that we use for posthog-js updates, you could use that

if CI passes and this is the only change we should be golden here

Reply:
Excellent call, let me add that real quick